src/test/java/com/jcabi/immutable/ArrayTest.java

```diff
@@ -197,4 +199,16 @@ public void makesSubList() throws Exception {
         );
     }
 
+    /**
+     * Changing array we passed to ctor doesn't affect immutable array content.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void implicitImmutable() throws Exception {
```

Comment:
@nesterov-n test name should be changed. See http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names